core/src/components/progress-bar/progress-bar.ios.vars.scss

```diff
@@ -0,0 +1,8 @@
+// iOS Progress bar
+// --------------------------------------------------
+
+/// @prop - Height of the progress bar
+$progress-bar-ios-height: 3px !default;
```

Comment:
Both in Figma and on the simulator, this appears to be 4px. What did you find that has a value of 3?

Reply:
This was bad refactoring on my part, should be `4px` 🤦